app/services/data/get-team-caseload.js

```diff
@@ -0,0 +1,27 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+module.exports = function (id) {
+  var table = 'team_caseload_overview'
```

Comment:
Think this view should be called `team_caseload_view`. This would also require an update to the view name in the seeds folder in `wmt-worker`